Userland/Libraries/LibAudio/GenericTypes.h

```diff
@@ -40,8 +40,8 @@ enum class ID3PictureType : u32 {
 // Note: This was first implemented for Flac but is compatible with ID3v2
 struct PictureData {
     ID3PictureType type {};
-    DeprecatedString mime_string {};
-    Vector<u32> description_string {};
+    String mime_string {};
+    String description_string {};
```

Comment:
Technically an unrelated change

Reply:
No, since we need the `read_from_stream` functions in String, which *are* the fix for the pointer mixup.